bin/selection_utils.py

```diff
@@ -164,55 +164,74 @@ def target_distance_fcn(args, this_seq, target_seqs):
 def calc_kd(node, args):
     if has_stop_aa(node.aa_seq):  # nonsense sequences have zero affinity/infinite kd
         return float('inf')
-    if not args.selection:
-        return 1.
-
-    assert args.mature_kd < args.naive_kd
-    tdist = node.target_distance if args.min_target_distance is None else max(node.target_distance, args.min_target_distance)
-    kd = args.mature_kd + (args.naive_kd - args.mature_kd) * (tdist / float(args.target_distance))**args.k_exp  # transformation from distance to kd
-
+    elif args.function_target_distance:
+        kd = args.mature_kd + (args.naive_kd - args.mature_kd) * (tdist / float(args.target_distance))**args.k_exp  # transformation from distance to kd
+    elif args.torchdms:
+        kd = 100 - 3*min(node.aa_seq.count('A'),33) ### arbitrary sequence to function mapping until we implement the torchdms api 
```

Comment:
I think we should wait to commit the torchdms stuff until it's implemented -- better to limit matsengrp/master to finished things.